docs/jade-getting-started.md

```diff
@@ -312,29 +246,170 @@ psql --list
+
+```
+./gradlew build           # build jade-data-repo and run unit tests
+./gradlew bootRun         # build jade-data-repo with Spring Boot features
+./gradlew check           # linters and unit tests
+```
+
+### 2. Run connected tests
+We don't recommend running the entire connected test suite locally, as it takes over an hour to run.
+Instead, you can select a specific test to run either in Intellij or the command line.
+First, make sure you have run through the following steps:
+* Start postgres
+* Ensure docker is running
+* You may need to re-auth with vault every so often. Run `vault login -method=github token=$(cat ~/.gh_token)`
+
+** Run test in the Command Line **
+* Run `source ./render-configs.sh` to pull secrets from vault and set environment variables
+* Run `./gradlew :testConnected --tests '*testCSVIngestUpdates'` to run a specific connected test
+
+** Run or Debug test in Intellij **
+* Run `./render-configs.sh -p` to pull secrets from vault and print environment variables
+* Set the environments printed out from the render configs script. You can do this either in
+your bash profile (you may need to then restart intellij) or in the intellij run configuration for the selected test.
+* Select test in intellij UI, select 'testConnected' and run or debug it
+
+### 3. Run Integration tests
+We don't recommend running the entire integrated test suite locally, as it takes over two hours to run.
+Instead, you can select a specific test to run either in Intellij or the command line.
+First, make sure you have run through the following steps:
+* Start postgres
+* Ensure docker is running
+* You may need to re-auth with vault every so often. Run `vault login -method=github token=$(cat ~/.gh_token)`
+
+** Run test in the Command Line **
+* Run `source ./render-configs.sh -a integration` to pull secrets from vault, set environment variables and point to Integration Synapse on Azure
+* Start the app locally with `./gradlew bootRun`
+* Open a new command line window, while bootRun runs in the background
+* In the new window, run `source ./render-configs.sh  -a integration` to pull secrets from vault and set environment variables in this context
+* Set this environment variable: `export IT_JADE_API_URL=http://localhost:8080`
+* Run `./gradlew :testIntegration --tests '*testSnapshotBuilder'` to run a specific integration test
+
+** Run or Debug test in Intellij **
+* Run `./render-configs.sh  -a integration -p` to pull secrets from vault and print environment variables
+* Set the environments printed out from the render configs script. You can do this either in
+  your bash profile (you may need to then restart intellij) or in the intellij run configuration for the selected test.
+  * Additionally, set these environment variables:
+  * `export IT_JADE_API_URL=http://localhost:8080`
+* Start application by running `./gradlew bootRun`
+* Select test in intellij UI, select 'testIntegration' and run or debug it
```

Comment:
Note for reviewers: This covers [DC-932 - Write instructions for getting on Azure on TDR - Running Azure integration tests locally](https://broadworkbench.atlassian.net/browse/DC-932).